src/Compute/Compute/help/New-AzVMConfig.md

```diff
@@ -446,7 +446,8 @@ Accept wildcard characters: False
 ```
 
 ### -Zone
-Specifies the availability zone list for the virtual machine. The allowed values depend on the capabilities of the region. Allowed values will normally be 1,2,3.
+Specifies the availability zone for the virtual machine. Although it takes in an array of zones, virtual machines do not support multiple availability zones.
+The allowed value depend on the capabilities of the region. Allowed value will normally be 1,2, or 3. More information on [Azure availability zones](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-overview#availability-zones).
 
```

Comment:
And we keep the 2 descriptions for this parameter the same across the two .md files?

As a user, if I see different descriptions I may be tempted to think that they are different things, when they are not.

Reply:
We do since they are different cmdlets. Currently the descriptions match.